doc/tools/stylelint.md

```diff
@@ -0,0 +1,17 @@
+# stylelint
+
+## install stylelint
+```sh
+  npm install --save-dev stylelint stylelint-config-standard
+```
+
+## how to play
+
+ Run stylelint and autofix:
+```sh
+  npx stylelint --fix "src/css/*.css"
```

Comment:
Tell the user he as to be in the same folder (where the npm install command was run)